src/main/java/alpha/Alpha.java

```diff
@@ -0,0 +1,144 @@
+package alpha;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+
+/**
+ * The Chatbot Alpha is capable of tracking, marking, listing, and deleting events, to-dos, and deadlines.
+ * It responds to commands like "event," "meeting," "deadline," "list," "mark," "unmark," and "delete."
+ * To end the Chatbot, type "bye."
+ *
+ * @author Wong Joon Hung
+ */
+public class Alpha extends Application{
+
+    private UI ui;
+    private FileHandler fileHandler;
+    private TaskList taskList;
+    private Parser parser;
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
+    /**
+     * Constructor for the Alpha class. It reads existing tasks from "data/alpha.txt" and adds them to the taskList.
+     */
+    public Alpha() {
+        ui = new UI();
+        fileHandler = new FileHandler();
+        taskList = fileHandler.readFromFile();
+        parser = new Parser(fileHandler, taskList, ui);
+        fileHandler.checkAndCreate();
+    }
+
+    /**
+     * Starts the Alpha ChatBot. Ends when "bye" is input.
+     */
+    /*
+    public void run() {
+        ui.introduce();
+        boolean isExit = false;
+        while (!isExit) {
+            String input = ui.read();
+            Command command = parser.parse(input);
+            command.execute();
+            isExit = command.isExit();
+        }
+        ui.goodbye();
+    }
+     */
```

Comment:
I think it would be better if you add a description on why you commented this code out. If there is no use for this method, then you can delete it entirely to reduce the amount of comments.